### DISCUSSION TITLE
Pull Request Preview Environments for increasing maintainer productivity

### DISCUSSION THREAD

**Discussion by waveywaves** [payloadcms/payload#1484] [General]
I would like to make life easier for Payload maintainers by implementing [Uffizzi](https://github.com/UffizziCloud/uffizzi) preview environments. 
Disclaimer: I work on [Uffizzi](https://github.com/UffizziCloud/uffizzi). 

Uffizzi is a Open Source full stack previews engine and our platform is available completely free for Payload (and all open source projects). This will provide maintainers with preview environments of their PRs in the cloud, allowing them iterate faster and reduce time to merge. 

Uffizzi is purpose-built for the task of previewing PRs and it integrates with your workflow to deploy preview environments in the background without any manual steps for maintainers or contributors.

TODO:
- [ ] Intial PoC

**Comment by denolfe** [comment]
This sounds like a great use-case for a [plugin](https://payloadcms.com/docs/plugins/overview).

I'm converting this to a discussion, as that is a better place for this.

**Reply by waveywaves** [reply]
Hey @denolfe we do have a plugin integration. Can I do a PoC with the integration if you would like that better ?

**Reply by denolfe** [reply]
Yes, the great thing about plugins is you can write one for anything 👍

**Reply by waveywaves** [reply]
Any chance we can get on a call so I can walk you through the plugin integration ?
